text/symbols.talon

```diff
@@ -1,5 +1,5 @@
 question [mark]: "?"
-(downscore | underscore): "_"
+(down score | under score): "_"
```

Comment:
Ok so I think the problem here is that these appear here and also implicitly via keys.py / keys.talon, with different spacing. This throws off talon's precedence rules. I would just remove this line

Also, given that this one was winning, I'd be tempted to use "underscore" in keys.py

@lunixbochs does that sound reasonable?

Reply:
Okay, that makes sense. I made this suggested change and that gets rid of the issue I was having as well as my original solution does. I see a few other duplications between this file and keys.py/keys.talon; is that something that should be cleaned up as well, or is there a reason to keep both around? Mostly just asking for curiosity's sake, though I'm happy to do more here if that is desired.

Reply:
Ah, of course. Yeah, we realistically shouldn't have duplicated command phrases.